.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,52 @@
+name: Daily build
+on:
+  schedule:
+    - cron: "0 1 * * *" # Every day at 1:00 AM
+  workflow_dispatch: {}
+jobs:
+  release_image:
+    name: Build and publish docker image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_REPOSITORY: mongodb/atlas
+      PLATFORMS: linux/amd64,linux/arm64
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set date
+        id: set-date
+        run: |
+          DATE=$(date +'%Y-%m-%d')
+          echo DATE=${DATE} >> $GITHUB_ENV
+      - name: 'Get latest tag'
+        id: get-latest-tag
+        run: echo "::set-output name=tag::$(git tag --list --sort=-version:refname "atlascli*" | head -n 1)"
+      - name: Extract version
+        run: |
+          release_tag=${{ steps.get-latest-tag.outputs.tag }}
+          echo "LATEST_VERSION=${release_tag#*/}" >> $GITHUB_ENV
```

Comment:
It is generally good pattern to keep steps self contained and visible. 
For bash scripts setting env vars we likely to benefit from shellcheck.

I advice we have those in scripts like here: https://github.com/mongodb/atlas-sdk-go/blob/main/.github/workflows/autorelease.yaml#L19

It will help us with:
- Maintenance
- Audit
- Self documentation of the release process.
- Testing outside github actions environment

Reply:
Noted and might look into this in a followup